.gitignore

```diff
@@ -11,4 +11,4 @@ src/agent/protocols/src/*.rs
 !src/agent/protocols/src/lib.rs
 build
 src/tools/log-parser/kata-log-parser
-
+.idea/
```

Comment:
If you intend on adding this, please add this as a separate commit as it is unrelated.